### PR TITLE
Add Role table,Required Column , Mail Validation

### PR DESCRIPTION
**Pull request by upadhyaydhiren**
You drop your schema then Run new code it automatically change into db.

### CODE REVIEW

File: src/main/java/com/utilaider/logistics/controller/LogisticsController.java [hunk 1]
```diff
@@ -87,4 +115,20 @@ public ModelAndView home(HttpServletRequest request, ModelMap map) {
...
+    Boolean isExistingEmail(@PathVariable("email") String email) {
+        try {
+            return ownerService.getOwnerByUsername(email) == null;
+        } catch (Exception e) {
+            e.printStackTrace();
+            return false;
+        }
+    }
+
+    @RequestMapping("reg")
```
Comment (shayaneumar) [inline src/main/java/com/utilaider/logistics/controller/LogisticsController.java:130]: Why not give fullname registration?

File: src/main/java/com/utilaider/logistics/utility/SecurityConfig.java [hunk 2]
```diff
@@ -26,9 +26,9 @@ public void configure(WebSecurity web) throws Exception {
 
     @Override
     protected void configure(HttpSecurity http) throws Exception {
-        http.addFilterBefore(new SimpleCORSFilter(), ChannelProcessingFilter.class)
+        http.csrf().disable().addFilterBefore(new SimpleCORSFilter(), ChannelProcessingFilter.class)
                 .authorizeRequests()
-                .antMatchers("/login/**", "/registration/**", "/logout/**","/test").permitAll()
+                .antMatchers("/login/**", "/registration/**", "/logout/**","/reg/**").permitAll()
```
Comment (shayaneumar) [inline src/main/java/com/utilaider/logistics/utility/SecurityConfig.java:31]: why is there 2 matchers registration and reg?

**Comment by shayaneumar** [comment]
closed

File: src/main/webapp/WEB-INF/jsp/Login.jsp [hunk 3]
```diff
@@ -94,5 +102,33 @@
                                 </section>
                             </div>
                         </div>
+                        <script type="text/javascript">
+                            function checkExistEmail()
+                            {
+                                var url = "http://locolhost:8080<c:url value="/existEmail/" />" + document.getElementById('email-coldRegistrationForm').value;
```
Comment (shayaneumar) [inline src/main/webapp/WEB-INF/jsp/Login.jsp:108]: http://locolhost:8080 don't do it like use relative path

File: src/main/java/com/utilaider/logistics/controller/LogisticsController.java (hunk 1 above)
Comment (upadhyaydhiren) [inline src/main/java/com/utilaider/logistics/controller/LogisticsController.java:130]: This method is not use any where I am Just Checking Registration jsp designed by you.

File: src/main/java/com/utilaider/logistics/utility/SecurityConfig.java (hunk 2 above)
Comment (upadhyaydhiren) [inline src/main/java/com/utilaider/logistics/utility/SecurityConfig.java:31]: In above comment I already said this is used testing of registration jsp. without permit I can't access that's why I mention reg in ant matchers.

File: src/main/webapp/WEB-INF/jsp/Login.jsp (hunk 3 above)
Comment (upadhyaydhiren) [inline src/main/webapp/WEB-INF/jsp/Login.jsp:108]: By mistake this line shown I always use relative path

**Pull request closed without merging**